indico/modules/events/contributions/controllers/management.py

```diff
@@ -496,6 +497,18 @@ def _process(self):
                                 contrib_types=contrib_types)
 
 
+class RHManageDefaultContributionDuration(RHManageContributionsBase):
+    """Dialog to manage the default contribution duration"""
+
+    def _process(self):
+        form = ContributionDefaultDurationForm(duration=contribution_settings.get(self.event, 'default_duration'))
+        if form.validate_on_submit():
+            contribution_settings.set(self.event, 'default_duration', form.duration.data)
+            flash(_("Default contribution duration was changed successfully"))
```

Comment:
added this flash, so that the user gets some sort of confirmation - maybe we need an error flash as well in case something goes wrong?

Reply:
No, because if:

- form validation fails, the dialog stays open and the error is shown next to the field
- something goes completely wrong, that's handled automatically with the error report dialog